tests/prov/test_multiple_config_deploy.py

```diff
@@ -60,6 +61,7 @@ def setup_class(cls):
                 cls.master_node_list.append(node_obj)
             else:
                 cls.worker_node_list.append(node_obj)
+        cls.log_disk_size = os.getenv('log_disk_size')
```

Comment:
for this we need to make change in Jenkins job

Reply:
Remove and keep the log disk size 4G.